### PR TITLE
feat(telecom): add heaset icon and several redirections

### PR DESCRIPTION
**Pull request by antonymarion**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? |no
| Tickets          | Feat#MANAGER-12919
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)

## Translation

- [x] CMT-7844

### CODE REVIEW

**Comment by JacquesLarique** [comment]
I think you forgot to add the menu entry for the accessories in the PNR version of the menu

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/line.controller.js [hunk 1]
```diff
@@ -105,7 +105,9 @@ export default /* @ngInject */ function TelecomTelephonyLineCtrl(
 
         promises.push(
           self.line.getPhone().then(() => {
-              self.displayMgcpBanner = this.isMgcpBannerAvailable() && self.line.phone.protocol === PHONE_PROTOCOL.MGCP;
+            self.displayMgcpBanner =
+              self.isMgcpBannerAvailable &&
```
Comment (JacquesLarique) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/line.controller.js:109]: Has isMgcpBannerAvailable became a variable ?

Comment (antonymarion) [reply packages/manager/apps/telecom/src/app/telecom/telephony/line/line.controller.js:109]: Yes, there was a bug I fixed directly here

**Comment by antonymarion** [comment]
TODO: @antonymarion update this dev since Chakra UI has been removed from Prod Manager code

**Review by qpavy — DISMISSED** [review]

File: packages/manager/apps/telecom/src/app/telecom/pack/dashboard/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -77,5 +77,6 @@
   "pack_xdsl_information_links": "Liens",
   "pack_xdsl_information_links_igmap": "Carte des incidents génériques",
   "pack_xdsl_information_links_travaux": "Travaux",
-  "pack_xdsl_information_links_guides": "Guides"
+  "pack_xdsl_information_links_guides": "Guides",
+  "line_order_accessories": "Commander un casque"
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/dashboard/translations/Messages_fr_FR.json:81]:
```suggestion
  "pack_xdsl_line_order_headset": "Commander un casque"
```

File: packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/voipLine.html [hunk 3]
```diff
@@ -21,6 +21,16 @@
         class="oui-tile__body"
         data-ng-if="!loaders.services && VoipLine.services.length > 0"
     >
+        <div class="oui-tile__body oui-tile__item_button">
+            <a
+                class="oui-button oui-button_icon-right oui-button_secondary oui-button_block"
+                data-ui-sref="telecom.orders.accessories"
+            >
+                <span data-translate="line_order_accessories"></span>
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/pack/slots/voipLine/voipLine.html:29]:
```suggestion
                <span data-translate="pack_xdsl_line_order_headset"></span>
```

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/container/src/public/translations/server-sidebar-order/Messages_fr_FR.json [hunk 4]
```diff
@@ -98,6 +98,7 @@
   "server_sidebar_item_telephony_prefix_trunk": "Trunk",
   "server_sidebar_item_sms_title": "SMS",
   "server_sidebar_item_fax_title": "Fax",
+  "server_sidebar_item_accessories_title": "Casque",
```
Comment (ghyenne) [inline packages/manager/apps/container/src/public/translations/server-sidebar-order/Messages_fr_FR.json:101]:
```suggestion
  "server_sidebar_item_headset_title": "Casque",
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/management/management.controller.js [hunk 5]
```diff
@@ -96,6 +96,14 @@ export default /* @ngInject */ function TelecomTelephonyLineManagementCtrl(
           'telephony_line_management_actions_line_to_number',
         ),
       },
+      {
+        name: 'line_order_accessories',
+        sref:
+          'telecom.telephony.billingAccount.line.dashboard.phone.accessories',
+        text: $translate.instant(
+          'telephony_line_phone_actions_line_order_accessories',
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/management/management.controller.js:104]:
```suggestion
          'telephony_line_phone_actions_line_order_headset',
```

File: packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -9,7 +9,7 @@
   "telephony_line_phone_actions_line_phone_reboot": "Redémarrer",
   "telephony_line_phone_actions_line_phone_order_phone": "Commander un téléphone VoIP",
   "telephony_line_phone_actions_line_phone_change_phone": "Changer de téléphone",
-  "telephony_line_phone_actions_line_order_accessories": "Commander des accessoires",
+  "telephony_line_phone_actions_line_order_accessories": "Commander un casque",
```
Comment (ghyenne) [inline packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/translations/Messages_fr_FR.json:12]:
```suggestion
  "telephony_line_phone_actions_line_order_headset": "Commander un casque",
```

**Review by ghyenne — DISMISSED** [review]

**Review by darsene — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by dectotam — DISMISSED** [review]

**Review by darsene — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by dectotam — APPROVED** [review]

**Pull request merged**